### ISSUE TITLE
iOS version check failed with iOS 10+

### ISSUE THREAD

**Issue by ltgbau** [kstenerud/ObjectAL-for-iPhone#98]
In iOS10+ version will always return 1, so AudioSession init wrong way

**Comment by KAMIKAZEUA** [comment]
Same here: 
Error: -[IOSVersion init]: Cannot parse iOS version string "10.0.1"

**Comment by EljoHB** [comment]
I've replaced it with the following code which seems to be just fine as only the major OS version is needed:

Edit the `IOSVersion.m` file:

`- (id) init`
`{`
	`if(nil != (self = [super init]))`
	`{`
`#ifdef __IPHONE_OS_VERSION_MAX_ALLOWED`
		`NSString* versionStr = [UIDevice currentDevice].systemVersion;`
		`NSArray* versionInfo = [versionStr componentsSeparatedByString:@"."];`
		`version = [versionInfo[0] intValue];`
`#else`
        `version = 5;`
`#endif`
	`}`

**Comment by KAMIKAZEUA** [comment]
I even removed using of this IOSVersion singleton, because now iOS 8 is minimum, so we don't need that old code for iOS 6 even /

**Comment by kstenerud** [comment]
I'll just remove it, then. No need to support ios 5 anyway.

**Comment by kstenerud** [comment]
07e1628d08002ec5246c357dec1561b609d8758a

**Comment by KAMIKAZEUA** [comment]
Yea, but in the latest Xcode minimum target is iOS 8 and who will compile for iOS 6 or what kind of device using it?

**Comment by kstenerud** [comment]
Doesn't matter. It's gone now.

**Comment by KAMIKAZEUA** [comment]
Hmm, so you think it's not need to update ObjectAL with latest iOS's because OpenAL been deprecated since iOS9? Looking forward for something new :)

**Comment by kstenerud** [comment]
Yeah, that'll come soon. But for now I'm just fixing the immediate problem :)

**Comment by kstenerud** [comment]
It turns out that OpenAL was not actually deprecated: https://forums.developer.apple.com/thread/7081

**Comment by KAMIKAZEUA** [comment]
OpenAL is listed as (Removed), so for SKD 9.0 probably will not be able to use it.
